composer.json

```diff
@@ -25,12 +25,20 @@
     "require": {
         "php": "^5.6 || ^7.0",
         "egeloen/ckeditor-bundle": "^4.0 || ^5.0",
-        "knplabs/knp-markdown-bundle": "^1.4",
+        "knplabs/knp-markdown-bundle": "^1.7",
```

Comment:
why this change?

Reply:
Because I was trying to reduce the matrix of possible options when doing prefer-lowest to have faster builds

Reply:
👍